packages/ext-angular-tests/src/app/test/TODO

```diff
@@ -0,0 +1,24 @@
+>>>Remaining components<<<
```

Comment:
Do you want this included? It also has React in it?

Reply:
@branflake2267  Yes, these are known issues that we are facing with respect to sencha tests.